Alexandr Antsiferov/lesson7/src/app/pages/User.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+import {fetchUser, resetUser} from'../actions/userAction';
+import { connect } from 'react-redux';
+import UserProfile from '../components/User';
+
+class User extends Component {
+
+  render() {
+    return (
+      <div>
+        {this.props.user ? <UserProfile {...this.props.user} /> : null}
```

Comment:
```suggestion
        <UserProfile {...this.props.user} />
```
`this.props.user` - это объект и он всегда будет `truthy` (даже если объект будет пустым/без свойств), т.е. преобразовываться в `true`